<?xml version="1.0"?>
<launch>
    <arg name="rigid_id" default="1"/>
    <arg name="ball_roi" default="0.6"/>

    <!-- run optitrack bridge -->
    <include file="$(find optitrack)/launch/optitrack_pipeline.launch">
      <arg name="iface" value="192.168.1.113"/>
    </include>

    <node name="bilbo_pose" pkg="ball_publisher" type="ball_publisher" output="screen">
        <param name="rigid_id" value="$(arg rigid_id)"/>
        <param name="ball_roi" value="$(arg ball_roi)"/>
    </node>
</launch>
